pylib/cqlshlib/cqlhandling.py

```diff
@@ -152,10 +151,10 @@ def cql_split_statements(self, text):
                     in_batch = True
         return output, in_batch or in_pg_string
 
-    def cql_complete_single(self, text, partial, init_bindings={}, ignore_case=True,
```

Comment:
Oof! I've been bit by this kind of thing in prod before...

Reply:
Yes, its a subtle and kinda nasty issue with the initializer is a static mutable value.